doozerlib/distgit.py

```diff
@@ -131,7 +131,7 @@ def clone(self, distgits_root_dir, distgit_branch):
                                      'local build will be sourced from the current dist-git '
                                      'contents and not the typical GitHub source. '
                                      )
-                    cmd_list = ["timeout", "900", "rhpkg"]
+                    cmd_list = ["timeout", str(self.runtime.global_opts['rhpkg_clone_timeout']), "rhpkg"]
```

Comment:
can this be de-inlined like the one below?
```suggestion
                     timeout = str(self.runtime.global_opts['rhpkg_clone_timeout'])
                     cmd_list = ["timeout", timeout, "rhpkg"]
```